### PR TITLE
quiet down the build

### PR DESCRIPTION
**Pull request by tpolecat**
This sets up the Travis build to run `sbt -q` with a custom logging config that only reports errors. (I verified that you do in fact see compile/test failures). I turned most of the `println`s that were in test output (I couldn't find all of them) into logger messages, so you still see them in a normal build and when running, but not in the CI log.

The CI log for this build should demonstrate silence. We'll see.

I also turned on caching which should speed things up a tiny bit. We'll see.

### CODE REVIEW

File: logging.properties [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+# To also add the FileHandler, use the following line instead.
+#handlers= java.util.logging.FileHandler, java.util.logging.ConsoleHandler
+
+# Default global logging level.
+# This specifies which kinds of events are logged across
+# all loggers.  For any given facility this global level
+# can be overriden by a facility specific level
+# Note that the ConsoleHandler also has a separate level
+# setting to limit messages printed to the console.
+.level= SEVERE
```
Comment (tpolecat) [inline logging.properties]: This is the only thing that is changed from the default config.

**Comment by tpolecat** [comment]
Oh wait now I know why we didn't do this before … it was timing out because there was no output. Probably need to add some hacking.

**Comment by tpolecat** [comment]
This seems to work. See the build transcript.

**Comment by cquiroz** [comment]
I guess it would blurp something if there is an actual error, right?

**Comment by cquiroz** [comment]
I wonder if we should include in travis building the specific modules supporting 2.12

**Comment by tpolecat** [comment]
> I guess it would blurp something if there is an actual error, right?

yes

> I wonder if we should include in travis building the specific modules supporting 2.12

good idea, i will add that

**Pull request merged**